packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -548,7 +552,7 @@ function removeCDUNK(payloadRaw: string): string {
 }
 
 function removeNullFlavor(payloadRaw: string): string {
-  const stringToReplace = /<id nullFlavor=".*?"/g;
-  const replacement = `<id extension="1" root="1"`;
+  const stringToReplace = /<id\s*nullFlavor\s*=\s*".*?"\s*\/>/g;
+  const replacement = `<id extension="1" root="1"/>`;
```

Comment:
Is this part of the scope of this ticket or fixing something we found in alerts/escalation/etc.?

Fine to have it here, it would just be good to have context about why this here so we can review it more easily. :)

Reply:
Weird this should be part of the ticket

Reply:
<img width="694" alt="Screenshot 2024-01-12 at 11 47 25 AM" src="https://github.com/metriport/metriport/assets/36802074/0b5de704-900b-4a22-a2c8-838be9ea4b7a">

So it seems like a github issue because if you go to the code editor view it doesnt show this